server/camcops_server/templates/base/base_web.mako

```diff
@@ -38,7 +38,9 @@ from camcops_server.cc_modules.cc_pyramid import (
 %>
 
 <%block name="body_tags">
-class="route_${ request.matched_route.name }"
+    %if getattr(request, "matched_route"):  ## may be e.g. None in PDF rendering
```

Comment:
Sorry! I actually didn't use this in the end so if it's more trouble than it's worth we could get rid of it.

Reply:
Generally I'd be indifferent but it's presumably a small performance hit on every page? If so, could just comment it out at this stage.